### ISSUE TITLE
Feature Request: Remove the front-end 2000 character limit of Bing Chat

### ISSUE THREAD

**Issue by nhan000** [adamlui/userscripts#43]
Currently the input box limit is 2000 characters, but apparently it's only a front-end limit and you can remove it using the Inspect Element tool ([Reddit](https://www.reddit.com/r/bing/comments/13l6dq4/comment/jkou33g/?context=3)) which has to be done again every time the page is reloaded or open a new tab.

Was wondering if you could cook up something that automatically does that for Bing. Thanks a lot in advance!

**Comment by nhan000** [comment]
btw, I remember you asked me about darkmode before. Currently Dark Reader doesn't work very well on Bing. I used this script https://greasyfork.org/en/scripts/462516-bing-chat-dark-mode for Bing Chat together with blocking the background with `www.bing.com##CIB-BACKGROUND` in the Filters in uBlockOrigin. Works really well.

**Comment by adamlui** [comment]
hey @nhan000 i tested and it lets you send msg even if you exceed the limit:

![image](https://github.com/adamlui/userscripts/assets/10906554/e8f2affc-ee58-4e30-89b7-278d5b403556)

Does it not work for you? Or do you mean only response doesn't work? If so, can you paste an example long query that works only with the reddit hack?

**Comment by nhan000** [comment]
Hi you can paste long inputs but you can't type text unless you edit the limit parameter. For example, try to type anything when you paste past the character limit in your example.

**Comment by adamlui** [comment]
@nhan000 ok i understand now! I am currently fixing my chatgpt script toggles to insert even when chat history is off, but after that I can begin working for you

**Comment by nhan000** [comment]
Thanks so much!

**Comment by nhan000** [comment]
Hey I just found this script that seems to work well [Remove Char Limit for Bing Chat AI](https://greasyfork.org/en/scripts/463347-remove-char-limit-for-bing-chat-ai). Sorry I did look up the script before opening this issue but I missed it.

This one [Enhanced Bing ChatAI](https://greasyfork.org/en/scripts/464747-enhanced-bing-chatai) seems to do a bit too much.

**Comment by adamlui** [comment]
I see, I will make one anyway because the code you provided is really convoluted & inefficient

**Comment by nhan000** [comment]
The hack I used has stopped working 😂

**Comment by adamlui** [comment]
@nhan000 is it the reddit hack?

**Comment by nhan000** [comment]
@adamlui yeah someone mentioned it on the edge Reddit, I think it's the same one as the one I sent you before ([Remove Char Limit for Bing Chat AI](https://greasyfork.org/en/scripts/463347-remove-char-limit-for-bing-chat-ai))

**Comment by adamlui** [comment]
@nhan000 ok I will figur an alternative method tomorrow, today I am porting ChatGPT Infinity to Chrome and Edge then eat then sleep
